src/tests/fixtures/frame_refresh_morph.html

```diff
@@ -1,3 +1,4 @@
 <turbo-frame id="refresh-morph">
   <h2>Loaded morphed frame</h2>
+  <h3>Does not change</h3>
```

Comment:
Adding a piece of HTML which does not change makes it easier to observe whether morphing is occurring or not within chrome inspector